File: src/main/java/de/zerotask/minecraft/tooltweaks/client/ClientProxy.java
package de.zerotask.minecraft.tooltweaks.client;

import de.zerotask.minecraft.tooltweaks.common.CommonProxy;
import de.zerotask.minecraft.tooltweaks.common.handler.TooltipHandler;
import net.minecraftforge.common.MinecraftForge;

public class ClientProxy extends CommonProxy {

    @Override
    public void init() {
        super.init();
        MinecraftForge.EVENT_BUS.register(new TooltipHandler());
    }
}

File: src/main/kotlin/de/zerotask/minecraft/tooltweaks/client/ClientProxy.kt
package de.zerotask.minecraft.tooltweaks.client

import de.zerotask.minecraft.tooltweaks.common.CommonProxy
import de.zerotask.minecraft.tooltweaks.common.handler.TooltipHandler
import net.minecraftforge.common.MinecraftForge

class ClientProxy: CommonProxy() {

    override fun init() {
        super.init()
        MinecraftForge.EVENT_BUS.register(TooltipHandler())
    }
}